map/traffic_manager.cpp

```diff
@@ -196,9 +196,13 @@ void TrafficManager::ThreadRoutine()
   {
     for (auto const & mwm : mwms)
     {
+      auto const & mwmInfo = mwm.GetInfo();
+      if (!mwmInfo)
+        continue;
+
       traffic::TrafficInfo info(mwm, m_currentDataVersion);
 
-      if (info.ReceiveTrafficData())
+      if (info.ReceiveTrafficData(m_trafficETags[mwmInfo->GetCountryName()]))
```

Comment:
Is it ok that in the calse of failure an empty string is put into m_trafficETags for the mwm?